src/guides/v2.3/config-guide/prod/config-reference-envphp.md

```diff
@@ -61,6 +62,18 @@ All the Magento cache types configuration are available from this node.
 
 Learn more about different [Cache Types][cache-types].
 
+## cron
+
+This node is used to enable/disable the crons. `0` means disable and `1` means enable. If crons are disabled, Jobs will not run.
```

Comment:
I'm not sure "crons" is a correct definition of the multiple jobs. Please, use "cron jobs" instead of "crons" since "crons" literally means multiple schedulers.